components/StyledListItem/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+const StyledListItem = styled.li`
+  width: 85%;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default value of the poperty, for this reason you do not need to set it.
```suggestion
```